README.NewResourceFromTerraform.md

```diff
@@ -488,6 +563,24 @@ samples are created for that resource.
 1.  Follow the sample guidelines [here](README.Samples.md) and create sample(s)
     for the resource.
 
+## Run/Disable Sample Tests
+
+In addition to the integration tests run using
+[pkg/test/testdata/resourcefixture](pkg/test/testdata/resourcefixture), KCC has
+tests for resource samples. These tests can be run manually as follows (example
+given is for one of CloudSchedulerJob's samples):
+
+```bash
+go test -v -tags=integration ./config/tests/samples/create -test.run TestAll -run-tests scheduler-job-pubsub
+```
+Replace the environment variables to real values before running the tests.
+
+Prow job will automatically run these tests in a newly created project. If this is
```

Comment:
We don't need to mention Prow job here (as it is internally CI/CD pipeline). If we want we can probably say The sample tests will be run periodically through internal CI/CD pipeline.